samples/snippets/src/test/java/com/example/spanner/SpannerSampleIT.java

```diff
@@ -68,6 +70,7 @@ public class SpannerSampleIT {
   private static final String encryptedDatabaseId = formatForTest(baseDbId);
   private static final String encryptedBackupId = formatForTest(baseDbId);
   private static final String encryptedRestoreId = formatForTest(baseDbId);
+  private static final long SECONDS_IN_ONE_DAY = TimeUnit.SECONDS.convert(24L, TimeUnit.HOURS);
```

Comment:
Does it make sense to rename `SECONDS_IN_ONE_DAY` to `STALE_INSTANCE_THRESHOLD_SECS`? In that case, if we need to change the number of hours to some other value in the future, we need not have to change this variable name.